File: C/struct_access_modify.c
//Access & modify structure members using pointer
#include <stdio.h>
#include <string.h>

struct Student {
    char name[50];
    int roll;
    float marks;
};

int main() {
    struct Student s = {"Ravi", 101, 88.5};
    struct Student *ptr = &s;

    printf("Original Name: %s\n", ptr->name);

    strcpy(ptr->name, "Kiran");  // Modify using pointer
    ptr->marks = 95.0;

    printf("Modified Name: %s, Marks: %.2f\n", ptr->name, ptr->marks);

    return 0;
}

File: Java/struct_access_modify.java
//Access & modify members using reference (object)
class Student {
    String name;
    int roll;
    float marks;
}

public class struct_access_modify {
    public static void main(String[] args) {
        Student s = new Student();
        s.name = "Ravi";
        s.roll = 101;
        s.marks = 88.5f;

        System.out.println("Original Name: " + s.name);

        // Modify using reference
        s.name = "Kiran";
        s.marks = 95.0f;

        System.out.println("Modified Name: " + s.name + ", Marks: " + s.marks);
    }
}
